src/app.config.js

```diff
@@ -43,7 +43,11 @@ const Config = {
     closed: 'closed',
     acknowledged: 'acknowledged'
   },
-  maxLogoFileSizeInBytes: 307200
+  maxLogoFileSizeInBytes: 307200,
+  deviceType: {
```

Comment:
Hmmm... this makes me want to move a few other things here. I hadn't seen these before (and those for ruleSeverity, alertStatus).